src/engraving/dom/rest.cpp

```diff
@@ -553,12 +611,40 @@ bool Rest::isWholeRest() const
 {
     TDuration durType = durationType();
     return durType == DurationType::V_WHOLE
-           || (durType == DurationType::V_MEASURE && measure() && measure()->ticks() < Fraction(2, 1));
+           || (isFullMeasureRest() && measure() && measure()->ticks() < Fraction(2, 1));
 }
 
-int Rest::computeNaturalLine(int lines) const
+bool Rest::isBreveRest() const
+{
+    TDuration durType = durationType();
+    return durType == DurationType::V_BREVE
+           || (isFullMeasureRest() && measure() && measure()->ticks() >= Fraction(2, 1));
+}
+
+int Rest::computeNaturalLine(DurationType type, int lines) const
```

Comment:
I would like to nuance some comments here a little bit, even though it's largely academic now at this point. 

In this case, it would be even better to make this a `static` method, decoupled from the Rest class. Why?
- makes the method _pure_, in the sense that it does not have any side effects and its output purely depends on which parameters you feed into it. I'm not going to explain the advantages of pure methods here in a GH comment, but if you've done a bit of functional programming (e.g. Haskell), you'll understand why I like them. (Sad that C++ does not _really_ have annotations for pure methods, similar to `const` and `noexcept`. That would be so awesome...)
- standalone methods are often more easily unit-testable. (That's not the whole story though; if a static method is private, or at the top-level inside a cpp file, you still can't test it of course.)